From gregkh@suse.de Thu May 15 21:04:50 2008
X-Envelope-From: <stable-commits-owner@vger.kernel.org>
X-Envelope-To: <vger@sl-h.de>
X-Delivery-Time: 1210881047
X-UID: 237678
Return-Path: <stable-commits-owner@vger.kernel.org>
X-RZG-CLASS-ID: mi
Received: from vger.kernel.org ([209.132.176.167])
	by mailin.webmailer.de (voltan mi32) (RZmta 16.35)
	with ESMTP id m07528k4FJcE1M for <vger@sl-h.de>;
	Thu, 15 May 2008 21:50:47 +0200 (MEST)
	(envelope-from: <stable-commits-owner@vger.kernel.org>)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1758947AbYEOTug (ORCPT <rfc822;vger@sl-h.de>);
	Thu, 15 May 2008 15:50:36 -0400
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S1758854AbYEOTuf
	(ORCPT <rfc822;stable-commits-outgoing>);
	Thu, 15 May 2008 15:50:35 -0400
Received: from cantor.suse.de ([195.135.220.2]:48585 "EHLO mx1.suse.de"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1758849AbYEOTue (ORCPT <rfc822;stable-commits@vger.kernel.org>);
	Thu, 15 May 2008 15:50:34 -0400
Received: from Relay1.suse.de (mail2.suse.de [195.135.221.8])
	(using TLSv1 with cipher DHE-RSA-AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mx1.suse.de (Postfix) with ESMTP id 5515E41A2B;
	Thu, 15 May 2008 21:50:33 +0200 (CEST)
Subject: patch usb-add-telstra-nextg-cdma-id-to-option-driver.patch added to 2.6.25-stable tree
To: gregkh@suse.de,
 captain.magnus@opensuse.org
Cc: <stable@kernel.org>,
 <stable-commits@vger.kernel.org>
From:	<gregkh@suse.de>
Date:	Thu, 15 May 2008 12:04:50 -0700
In-Reply-To: <200805141921.m4EJLMTw020032@hera.kernel.org>
Message-Id: <20080515195032.994B614540CB@imap.suse.de>
Sender:	stable-commits-owner@vger.kernel.org
Precedence: bulk
Reply-To: linux-kernel@vger.kernel.org
List-ID: <stable-commits.vger.kernel.org>
X-Mailing-List:	stable-commits@vger.kernel.org
Status: R
X-Status: NC
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  


This is a note to let you know that we have just queued up the patch titled

     Subject: USB: add Telstra NextG CDMA id to option driver

to the 2.6.25-stable tree.  Its filename is

     usb-add-telstra-nextg-cdma-id-to-option-driver.patch

A git repo of this tree can be found at 
    http://www.kernel.org/git/?p=linux/kernel/git/stable/stable-queue.git;a=summary


>From stable-bounces@linux.kernel.org Wed May 14 12:21:45 2008
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 14 May 2008 19:21:22 GMT
Subject: USB: add Telstra NextG CDMA id to option driver
To: jejb@kernel.org, stable@kernel.org
Message-ID: <200805141921.m4EJLMTw020032@hera.kernel.org>

From: Greg Kroah-Hartman <gregkh@suse.de>

commit 23cacd65f65956426bbca25964a68c174db83a31 upstream

As reported by Magnus Boman <captain.magnus@opensuse.org>

Cc: Magnus Boman <captain.magnus@opensuse.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/option.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -299,6 +299,7 @@ static struct usb_device_id option_ids[]
 	{ USB_DEVICE(KYOCERA_VENDOR_ID, KYOCERA_PRODUCT_KPC680) },
 	{ USB_DEVICE(QUALCOMM_VENDOR_ID, 0x6613)}, /* Onda H600/ZTE MF330 */
 	{ USB_DEVICE(MAXON_VENDOR_ID, 0x6280) }, /* BP3-USB & BP3-EXT HSDPA */
+	{ USB_DEVICE(0x19d2, 0x0001) }, 	/* Telstra NextG CDMA */
 	{ } /* Terminating entry */
 };
 MODULE_DEVICE_TABLE(usb, option_ids);


Patches currently in stable-queue which might be from gregkh@suse.de are

queue-2.6.25/block-do_mounts-accept-root-non-existant-partition.patch
queue-2.6.25/powerpc-bolt-in-slb-entry-for-kernel-stack-on-secondary-cpus.patch
queue-2.6.25/usb-remove-picdem-fs-usb-demo-device-from-ldusb.patch
queue-2.6.25/usb-unusual_devs-add-support-for-gi-0401-sd-card-interface.patch
queue-2.6.25/usb-option-add-new-dell-5520-hsdpa-variant.patch
queue-2.6.25/usb-add-zoom-telephonics-model-3095f-v.92-usb-mini-external-modem-to-cdc-acm.patch
queue-2.6.25/usb-do-not-handle-device-1410-5010-in-option-driver.patch
queue-2.6.25/usb-add-telstra-nextg-cdma-id-to-option-driver.patch
queue-2.6.25/memory_hotplug-always-initialize-pageblock-bitmap.patch
queue-2.6.25/ext3-4-fix-uninitialized-bs-in-ext3-4_xattr_set_handle.patch
queue-2.6.25/libata-force-hardreset-if-link-is-in-powersave-mode.patch
--
To unsubscribe from this list: send the line "unsubscribe stable-commits" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

